server/views/patterns/typography.njk

```diff
@@ -123,257 +121,254 @@ p q r s t u v w x y z { | } ~
   </div>
 
   <h3>Figures</h3>
-  <p>Figures are usually used to refer to images:</p>
+
+  <h4>Image-based figure</h4>
   <div class="example">
     <figure>
-      <img src="http://lorempixel.com/680/400/abstract" alt="Example image" />
+      <img style="max-width: 100%;"
```

Comment:
Image with caption, I image would have an image component in it no? To stop having to write this out leaving loads of room for forgetting something.

Nice.